04_HowTos/QueryClient/How_to_use_the_Data_Lab_QueryClient.ipynb

```diff
@@ -294,39 +315,420 @@
-    "results = qc.query(adql=query,async_=True,wait=True,poll=15,verbose=1)"
+    "print(qc.mydb_list(),\"\\n\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now let's drop the table from our MyDB."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": 8,
    "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "'OK'"
+      ]
+     },
+     "execution_count": 8,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "qc.mydb_drop('zxmags')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Ensure it has been removed by calling the *mydb_list()* function again:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "c4d,created:2020-08-27 14:55:07 MST\n",
+      "cd,created:2020-08-27 15:08:40 MST\n",
+      "mags,created:2020-08-28 15:26:07 MST\n",
+      "newermags,created:2020-08-28 15:49:49 MST\n",
+      "quickresults,created:2020-08-25 14:44:34 MST\n",
+      "quickresults2,created:2020-08-25 14:46:01 MST\n",
+      "usno_objects,created:2020-08-25 12:10:18 MST\n",
+      "usno_test,created:2020-08-20 12:21:28 MST\n",
+      " \n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(qc.mydb_list(),\"\\n\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a class=\"anchor\" id=\"importmydb\"></a>\n",
+    "# Import data into a MyDB table with QueryClient\n",
+    "\n",
+    "Users can use this function to import data saved on a local computer or import data from VOSpace into a MyDB data table. The data must be in the form of either a CSV file or Pandas Dataframe object in order to load it into MyDB.\n",
+    "\n",
+    "First let's query some data from the Data Lab database and save it locally as a CSV file:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "'OK'"
+      ]
+     },
+     "execution_count": 10,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "query = \"select * from gaia_dr1.gaia_source limit 10\"\n",
+    "qc.query (adql=query, fmt='csv', out='./gaiaresult.csv')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Next we will use the *mydb_import* function to import the locally stored CSV file into a MyDB table:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "'OK'"
+      ]
+     },
+     "execution_count": 11,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "qc.mydb_import('resultname','./gaiaresult.csv')"
```

Comment:
Wouldn't it be better if the table name in MyDB was more descriptive than 'resultname'?